src/main/java/com/tmh/controller/admin/UserController.java

```diff
@@ -17,10 +23,72 @@
 	private static final Logger logger = Logger.getLogger(UserController.class);
 
 	@RequestMapping(value = "/users")
-	public String userList(Model model) {
-		logger.info("user list page");
+	public String showUserList(Model model) {
+		logger.info("show users list");
+		
 		model.addAttribute("users", userService.findAll());
+		
 		return "views/admin/userManager/userList";
 	}
 	
-}
\ No newline at end of file
+	@RequestMapping(value = "/users/{id}")
+	public String showUserDetail(@PathVariable("id") int id, Model model) {
+		logger.info("show user detail");
+		
+		User user = userService.findById(id);
+		
+		if (user == null) {
+			model.addAttribute("css", "danger");
+			model.addAttribute("msg", "User Not Found");
```

Comment:
` "User Not Found"` chú khai báo và tập sử dụng messages.properties hì

key để tìm là ` spring mvc I18n` nhé

Reply:
Vâng chị ơi !! Em sẽ tìm hiểu ạ :))